profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -298,26 +296,27 @@ function _nexteuropa_multilingual_alias_node_form_alter(&$form, &$form_state) {
  *   - alias: The the LANGUAGE_NONE alias.
  *   - pathauto: The flag indicating if pathauto is activated for the node.
  */
-function _nexteuropa_multilingual_get_und_entity_alias_info($entity, $entity_type) {
+function _nexteuropa_multilingual_get_und_entity_alias_info($entity_type, $entity) {
```

Comment:
Yes, I realized that this code I've done in a previous ticket was not aligned on the majority of API existing in Drupal and Entity module. 
Then, I aligned the function signature on what I observed to ease the code reading. As it is a private function and it is only used once so far, risks related to execution time errors are in existent.